org.eclipse.xtext.common.types.eclipse.tests/src/org/eclipse/xtext/common/types/xtext/ui/CommonTestLanguagesFragments.mwe2

```diff
@@ -9,36 +9,40 @@ module org.eclipse.xtext.common.types.xtext.ui.CommonTestLanguagesFragments
 
 import org.eclipse.xtext.generator.*
 import org.eclipse.xtext.ui.generator.*
+import org.eclipse.xtext.xtext.generator.*
+import org.eclipse.xtext.xtext.generator.ui.*
+import de.itemis.xtext.antlr.XtextAntlrGeneratorFragment
+import de.itemis.xtext.antlr.XtextAntlrUiGeneratorFragment
 
-var referencedGenModels =""
-var genModel = ""
-var basePackage = ""
-var javaModelDirectory = ""
-var xmiModelDirectory = ""
-var file.extensions = ""
-var classSplitting = true
-var skipGenerate = false
-var backtrack = false
-var memoize = false
-var emfRuntimeVersion = "2.9"
-
-CompositeGeneratorFragment {
-	fragment = ecore.EcoreGeneratorFragment auto-inject {}
-	fragment = grammarAccess.GrammarAccessFragment auto-inject {}
-	fragment = resourceFactory.ResourceFactoryFragment { 
-		fileExtensions = file.extensions
-	}
-	fragment = serializer.SerializerFragment auto-inject {
-		srcGenOnly = true
-	}
-	fragment = scoping.ImportNamespacesScopingFragment {}
-	fragment = exporting.QualifiedNamesFragment {}
-	fragment = types.TypesGeneratorFragment {}
-	fragment = parser.antlr.XtextAntlrGeneratorFragment auto-inject {
+CompositeGeneratorFragment2 {
```

Comment:
Should keep old name

Reply:
Is this fragment even used somewhere?

Reply:
????? the concept of nested fragments seems different in new workflow. this is a java class with that name that composes this fragments

Reply:
My bad. Sorry